src/Security/LiFiTimelockController.sol

```diff
@@ -26,15 +27,32 @@ contract LiFiTimelockController is TimelockController {
     /// @param _minDelay Initial minimum delay for operations
     /// @param _proposers Accounts to be granted proposer and canceller roles
     /// @param _executors Accounts to be granted executor role
+    /// @param _deployerWallet Address of the deployer wallet (that will be granted CANCELLER role)
     /// @param _admin The address that will be the admin of the TimelockController (= the LI.FI MultiSig SAFE)
+    /// @param _diamond The address of the diamond contract that this timelock controls
     constructor(
         uint256 _minDelay,
         address[] memory _proposers,
         address[] memory _executors,
+        address _deployerWallet,
```

Comment:
can we name it '_canceller' or '_cancellerWallet'?

Reply:
good point, changed.
I also added docs and deploy scripts (they were still in another open PR)